client/packages/invoices/src/InboundShipment/DetailView/modals/InboundLineEdit/TabTables.tsx

```diff
@@ -56,6 +60,52 @@ const getExpiryColumn = (
     backgroundColor: alpha(theme.palette.background.menu, 0.4),
   },
 ];
+const getNumberOfPacksColumn = (
+  updateDraftLine: (patch: Partial<DraftInboundLine> & { id: string }) => void,
```

Comment:
I think it would be better to add a `required` prop to the `NonNegativeIntegerCell`, or even lower, in the `BasicTextInput`. That way it could be shared by other things too

Originally I thought that the asterisk should always show, like a 'required' field indicator, but I like the conditional version that you have.. it's more helpful!


I've put my changes into a branch, so you can see what I mean: https://github.com/openmsupply/open-msupply/compare/%231051-When-entering-a-new-shipment-line%2C-alert-if-mandatory-fields-are-not-filled-in%2C-rather-than-just-closing-the-window...1051-number-of-packs-cell